source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -333,7 +334,20 @@ def _getColumnContentRaw(self, index):
 		return buffer.value if buffer else None
 
 	def _getColumnContent(self, column):
-		return self._getColumnContentRaw(self.parent._columnOrderArray[column - 1])
+		targetColumn = self.parent._columnOrderArray[column - 1]
+		try:
+			return self._getColumnContentRaw(targetColumn)
+		except ArgumentError:
+			# In case that a 64-bit pointer to the text had been  returned /rjh

```

Comment:
Initials are usually not used in comments. If it is required to contact the comments author, we can use git blame for that.

However, it might be good to point at the issue you reported. e.g. `# #8175: `# In case that a 64-bit pointer to the text had been  returned, use the display text`